Expensify/App
# Anthony  toggle show password

### Details
Added a toggle button icon to password fields to change visibility

### Fixed Issues
https://github.com/Expensify/App/issues/2734

### QA Steps
Web and Desktop:
1. Type something into a password field
2. Verify password is hidden
2. Click eye icon
3. Verify the password is visible
4. Click again
5. Verify it is hidden again

Android and iOS
1. Type something into a password field
2. Verify password is shown
2. Click eye icon
3. Verify the password is hidden
4. Click again
5. Verify it is shown again

### Tested On


### Screenshots
setpassword screen web:
https://user-images.githubusercontent.com/47184118/129957727-cff28fb8-f9f5-4d6c-9d69-cd469e2e74e7.mp4

#### Edge
![image](https://user-images.githubusercontent.com/47184118/133485662-792a5f52-9ee8-45e8-958f-6864d1cfc7b5.png)
note the lack of vendor specific show password functionality

#### Mobile Web
![image](https://user-images.githubusercontent.com/47184118/133486050-3217571d-bda2-46b6-bf3f-92a4fb11fa8c.png)

#### Desktop
https://user-images.githubusercontent.com/47184118/133485179-aa45ff31-6457-4865-a207-6eff4177ae66.mov

#### iOS
![image](https://user-images.githubusercontent.com/47184118/133147803-331030a5-5c20-426a-990b-d8e93511b770.png)

#### Android
![image](https://user-images.githubusercontent.com/47184118/133484801-bbc986b8-9293-408b-b722-7b9f7c9f0fa7.png)
